SCIM_resource_type.json

```diff
@@ -1,60 +1,60 @@
 [
   {
     "schemas": ["urn:ietf:params:scim:schemas:core:2.0:ResourceType"],
-    "id": "Device",
-    "name": "Device",
-    "endpoint": "/Device",
-    "description": "Device Account",
-    "schema": "urn:ietf:params:scim:schemas:core:2.0:Device",
+    "id": "Devices",
+    "name": "Devices",
+    "endpoint": "/Devices",

```

Comment:
I believe only this needs to be pluralized. 
For reference, this is how it is done for the User and Group resource types: https://datatracker.ietf.org/doc/html/rfc7643#section-8.6

The endpoint is `/Users` and `/Groups` respectively but is singular everywhere else.